### ISSUE TITLE
Building from the source is giving lots of errors, references missing, but I could still build

### ISSUE THREAD

**Issue by eriawan** [OpenLiveWriter/OpenLiveWriter#132]
Hi guys,

I have forked and clone the repo of OpenLiveWriter and I opened the `writer.sln` as suggested by the building guide page on Wiki. 
My intention is to understand the code by and examine the projects and the codes. 
After I have opened the solution and I rebuild it, there are lots of red squiggles on the source codes. The squiggles describe errors about some namespaces don't exist, even some method/types couldn't be found, like the error in `BlogRunner\Program.cs`

![openlivewriter_repo_vs2015_error](https://cloud.githubusercontent.com/assets/8773147/11788060/9a92a4ce-a2c1-11e5-8132-b08281de63f6.png)

I have tried to clean and rebuild it, and the errors are there.

Also there are references to "Analyzer" but the references are not available. Is this "Analyzer" build in from Visual Studio, Roslyn, or external? Please provide advice.

My machine is: Windows 10 Professional 1511 with Visual Studio 2015 Update 1 fresh install.

**Comment by hashhar** [comment]
Some things I would like you to check:
1. Did you load all the projects in the solution file or only some of them?
2. Did you use Nuget Package Manager to do Package Restore?
3. Check for updates to .NET in Windows Update.

**Comment by ScottIsAFool** [comment]
Have you run .\build.cmd in the root folder of the repository? Make sure you've done as the instructions state https://github.com/OpenLiveWriter/OpenLiveWriter/wiki/Building,-Testing,-and-Debugging

**Comment by kathweaver** [comment]
I've not had any problems building the code.  I did make sure that I had updated Visual Studio with all the latest updates before I tried to build.  

However, I didn't do a fork or clone -- I downloaded the zip file and extracted it.

**Comment by kathweaver** [comment]
Okay, I used the fork and clone method of obtaining the source, and again compiled and ran with no issues other than having to set the correct project.

**Comment by kathweaver** [comment]
This is what my Visual Studio looks like:

<img width="972" alt="image" src="https://cloud.githubusercontent.com/assets/7053084/11796891/ba9295a0-a284-11e5-8ad9-5543876b05bd.png">

**Comment by RobStand** [comment]
@eriawan Are you still having issues building OLW?

**Comment by eriawan** [comment]
@hashhar,

I'm using Windows 10 Build 1511. It's already the latest version of Windows 10, and currently on this date, there's no further update for my .NET 4.6.1 on Windows 10. Also I have tried Nuget package restore, and the packages restored successfully, and I also load all of the projects in the `writer.sln` solution.

@kathweaver ,

I'm using a fresh install of VS 2015 Update 1. I guess you're not using the same version of Visual Studio 2015 as mine, because the feedback icon is different from the icon in VS 2015 Update 1. It can be problems on my machine. Could you help me to check to open OLW solution in VS 2015 Update 1?

@RobStand ,

I'm still having errors and squiggles on the OLW project. I have tried and installed it on VM with the same VS 2015 Update 1, and the errors and squiggles are there also.

@ScottIsAFool ,
I can run the `build.cmd` script successfully, but still my main issue is the way the project is opened is still giving errors. Again, I can also compile the project successfully using either clean->rebuild or simply rebuild.

@RobStand, @ScottIsAFool, have you guys tried to open OLW solution in VS 2015 Update 1? I'm curious about this matter/issue.

**Comment by hashhar** [comment]
I am using VS2015 Update 1 myself. This is peculiar.

**Comment by eriawan** [comment]
Guys,

I have found a quick fix for this using these steps: 
1. delete all of the restored nuget packages
2. delete all of the files in the i386 build folder
3. restore the nuget packages
4. force clean and rebuild for the solution

Now the project is opened without errors, especially squiggles.

Many thanks to @kathweaver, @ScottIsAFool, and @RobStand for helping me before! :+1: 

I have tested and tried to reproduce the issue on a VM, and to my surprise I was able to reproduce my issue. 

It seems VS 2015 Update 1 has trouble when opening large solution that has many projects in it.
FYI: when opening OLW in VS 2015, my issue did not occur at all.

I'll close this issue.
